docs/characterization/isolated_xeb.ipynb

```diff
@@ -208,6 +208,103 @@
     "plt.xlabel('Cycle Depth $d$')\n",
```

Comment:
<p>The same comment as in parallel version (describe options better)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3795/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>done</p>